pyo3-bytes/src/bytes.rs

```diff
@@ -327,9 +330,13 @@ impl PyBytes {
 
 impl<'py> FromPyObject<'py> for PyBytes {
     fn extract_bound(ob: &Bound<'py, PyAny>) -> PyResult<Self> {
-        let buffer = ob.extract::<PyBytesWrapper>()?;
-        let bytes = Bytes::from_owner(buffer);
-        Ok(Self(bytes))
+        if ob.is_none() {
```

Comment:
We shouldn't automatically accept `None` for PyBytes. Downstream users can explicitly opt-in to accepting `None` by using `Option<PyBytes>` in their type

Reply:
boom done